presto-product-tests/src/main/java/io/prestosql/tests/hive/TestSyncPartitionMetadata.java

```diff
@@ -22,13 +22,16 @@
 import io.prestosql.tempto.query.QueryResult;
 import org.testng.annotations.Test;
 
+import java.sql.SQLException;
```

Comment:
This looks unused

Reply:
It's used in an assertion below (https://github.com/prestosql/presto/pull/3830/files#diff-6536cdd5dd9e669a967fccf854e75924R124).